nss-day-cohort-39/hello-world-the-globetrotters
# Iceland Page Basic Info Setup

### Changes
- Setup basic iceland info with no styling.
- Picture needs formatting but is present

### Testing
- Make sure iceland info is present in three detail box
- Make sure picture is loading
